Euroleague-Graduate/Euroleague-Graduate/TurkishAirlinesApiClient.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TurkishAirlinesAPIClient.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/19/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
```

Comment:
This generic api client, name should be ApiClient or EuroleagueApiClient... 
as it shouldn't be related to any specific league